cves/kernel/CVE-2020-26088.yml

```diff
@@ -452,7 +470,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    I believe that the mistake in this scenario was the idea that processes could leverage this lack of privilege check to create raw
```

Comment:
This is not really the mistake that was made but a symptom of the mistake

Reply:
I see where you are coming from, but I do think that part of the mistake was the developers failing to realize that a privilege check was necessary here, thus not including the check. The first line just sort of expands on that failed realization.